sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/TextAnalyticsSubscriptionKeyCredential.java

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.textanalytics.models;
+
+/**
+ * Subscription key credential that shared across cognitive services, or restrict to single service.
```

Comment:
Is this applicable for all cognitive services? If so, is the plan to duplicate this credential in all cognitive services modules or would we create a common module like storage has to put the common stuff there?

Reply:
We had a discussion about having a common module. We decided not to have a common module now because currently we only have one service implemented. We don't really know the use case.

Reply:
@mssfang is right - this is still an open question.  We're starting to look at FormRecognizer now, though, so we'll start driving the "where would we put a common Cognitive credential" with the architects.